routes/cube_routes.js

```diff
@@ -4174,4 +4174,20 @@ router.get(
   }),
 );
 
+router.get('/api/date_updated/:id',
+  util.wrapAsyncApi(async (req, res) => {
+    const { id } = req.params;
+    const cube = await Cube.findOne(buildIdQuery(id)).lean();
```

Comment:
you can make this more performant by doing:
`await Cube.findOne(buildIdQuery(id), 'date_updated').lean();`

This will only fetch the only the `date_updated` field. If you need other fields, just add them to the string, separated with spaces